client/gutenberg/extensions/map-block/add-point/oval.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This SVG can be cleaned up and minified.

Reply:
Since the raster assets are going to be in the WP.com CDN, I opted to move this one too in https://github.com/Automattic/wp-calypso/pull/27836/commits/c778a244c2702e3e96b782aa062d8c96e243f57c. Currently it is in the Atavist S3 but will be moved to WP.com shortly.